main.py

```diff
@@ -58,9 +58,43 @@ def upload_file(
     return response
 
 
-@app.get("/bees/{file_path:path}")
-def fetch_bee_photo(file_path: str):
-    return {"file_path": file_path}
+@app.get("/download_bee/")
+def download_bee_photo(
+    bee_photo_name: str, bucket: Optional[str] = None, acl: Optional[str] = None
+) -> str:
+    """
+    Download a bee photo from an S3 bucket
+    - **bee_photo_name**: required, str including file extension
+    - **bucket**: the S3 bucket to post the file object to
+    - **acl**: the S3 access control list type
+    Returns a dictionary with the path of the file you downloaded
+    """
+    response = download_file(file_name=bee_photo_name, bucket=bucket, acl=acl)
```

Comment:
depending the response of the helper function we can also raise a FastAPI exception here if file is not found, e.g.

```
raise HTTPException(status_code=404, detail=f"{bee_photo_name} not found.")
```